examples/plot_survival_analysis.py

```diff
@@ -195,11 +195,74 @@
 ax.set_ylabel("objective suboptimality")
 _ = ax.set_xlabel("time in seconds")
 
-
-
 # %%
 # According to printed ratio, using ``skglm`` we get the same result as ``lifelines``
 # with more than x100 less time!
 speed_up = records["lifelines"]["times"][-1] / records["skglm"]["times"][-1]
 print(f"speed up ratio: {speed_up:.0f}")
 
+# %%
+# Efron estimate
+# ==============
+#
+# The previous results, namely closeness and timings, can be extended to the case
+# of handling tied observation with the Efron estimate.
+#
+# Let's start by generating data with tied observation. This can be achieved
+# by passing in a ``with_ties=True`` to ``make_dummy_survival_data`` function.
+tm, s, X = make_dummy_survival_data(
+    n_samples, n_features,
+    normalize=True,
+    with_ties=True,
+    random_state=0
+)
+
+# check data has tied observations
+print("Number of unique times", np.unique(tm))
```

Comment:
If it can fit on a single line with "number of unique times : xx out of yy samples" it's nicer

Reply:
Looks much cleaner, thanks!